spec/models/budget_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Budget, type: :model do
+  before(:each) do
+    @user = User.create(name: 'Elie', email: 'test@gmail.com', password: 'password')
+    @group = Group.create(user: @user, name: 'Food', icon: 'https://icon_url')
+    @budget = Budget.create(user: @user, name: 'Chips', amount: 15)
+  end
+
+  scenario 'it should be valid' do
+    expect(@budget).to be_valid
+  end
+end
```

Comment:
- Please ensure all test cases are passing for your app 👍🏽. Some are still failing kindly fix them 😉 
![Screenshot from 2023-02-23 20-43-36](https://user-images.githubusercontent.com/44624138/221014954-e7b2c238-6442-4bb3-8a8b-f4fccd24c922.png)